src/main/resources/antisamy.xml

```diff
@@ -795,9 +796,9 @@ http://www.w3.org/TR/html401/struct/global.html
 				</regexp-list>
 			</attribute>
 			<attribute name="rel">
-				<literal-list>
-					<literal value="nofollow"/>
-				</literal-list>
+				<regexp-list>
+					<regexp value="(\s*(nofollow|noopener|noreferrer)(\s(nofollow|noopener|noreferrer))*\s*)?"/>
```

Comment:
Does this regex allow duplicate entries? If so, it there a way to create a regex that allows these 3, but no duplicates of them?

Reply:
It does allow duplicates. The only way to allow the three words with no duplicates regardless the order is to specify a regex with all possible combinations which would look long and ugly. That's why I left it to at least have the valid words with compulsory spaces (if there are more than one).